pkg/controller/miganalytic/volume_adjustment_test.go

```diff
@@ -270,11 +270,11 @@ func TestPersistentVolumeAdjuster_calculateProposedVolumeSize(t *testing.T) {
 				actualCapacity:    resource.MustParse("200"),
 				requestedCapacity: resource.MustParse("20"),
 			},
-			wantProposedSize: resource.MustParse("200"),
+			wantProposedSize: resource.MustParse("106"),
```

Comment:
I see that 103% is desired but `wantProposedSize: "106"`, is this a typo?

Reply:
Yes, a typo and a logic flaw :)

Reply:
Ohh now that I realized, this is not a logic issue. 106 is the correct value, the reference for usage percentage is the actual volume size which is 200 in this case, which makes the 3% = 6Gi buffer.